gateway-server/src/main/java/org/apache/knox/gateway/services/token/impl/DefaultTokenStateService.java

```diff
@@ -208,6 +205,18 @@ protected void updateExpiration(final String token, long expiration) {
     }
   }
 
+  protected void removeRevokedExpiredToken(final String token) {
```

Comment:
Again, removeToken(String) should be a sufficiently-clear name.

Reply:
Addressed in new update.